AddressParser/AddressParser.cs

```diff
@@ -476,6 +482,9 @@ public string StatePattern
         /// </value>
         public string StreetPattern => @"
                         (?:
+                          # special case for addresses like 100 COUNTY ROAD F  http://regexstorm.net/tester?p=%28%3f%3a%28%3f%3cSTREET%3eCOUNTY%5cW%2bROAD%5cW*%5cw%2b%29%29&i=COUNTY+ROAD+FG%0d%0a%0d%0a&o=ixs
+                          (?:(?<STREET>COUNTY\W*ROAD\W*\w+))
+                          |
```

Comment:
There is an issue with this regex now that I've tested it against the 10k sample set.  Consider these two addresses,

713 COUNTY ROAD N, EDGAR        , WI 54426
3716 COUNTY ROAD E, SPENCER      , WI 54484
1844 COUNTY ROAD S, EDGAR        , WI 54426

Because S, N and E are directionals for South, North and East, the new regex seems to cause them to be picked up as post directionals instead of the road name.  There are no examples for W in the test set, but presumably, they would have the same problem.

Old regex was parsing fine (probably because the comma was helping it distinguish where the road ended.)